clang/lib/Format/FormatToken.h

```diff
@@ -403,6 +412,17 @@ struct FormatToken {
   /// \c true if this token ends a group of C++ attributes.
   unsigned EndsCppAttributeGroup : 1;
 
+private:
+  /// Kind of block comment.
+  CommentKind BlockCommentKind = CommentKind::Plain;
+
+public:
+  CommentKind getBlockCommentKind() const { return BlockCommentKind; }
+  void setBlockCommentKind(CommentKind Kind) {
+    BlockCommentKind = Kind;
+    assert(getBlockCommentKind() == Kind && "CommentKind overflow!");
```

Comment:
Since you don't use just a few bit, but the full enum, you don't need that assert.